index.html

```diff
@@ -43,6 +43,105 @@ <h2 class="jobtitle">Software Developer</h2>
+                <img src="image/css-divider.png" alt="divider image" class="subgrid-item">
+                <a href="#" class="subgrid-item">bootstrap</a>
+                <img src="image/css-divider.png" alt="divider image" class="subgrid-item">
+                <a href="#" class="subgrid-item">Ruby</a>
+              </div>
+              <a href="#" class="grid-item seeproject">See Project</a>
+            </div>
+
+            <div class="project-grid">
+              <div class="grid-item"><img src="image/Img-Placeholder.png" alt="image of project"></div>
+              <div class="grid-item"><h2>Multi-Post Story</h2></div>
+              <div class="grid-item holdtext">
+                A daily selection of privately personalized reads; 
+                no accounts or sign-ups required. 
+                has been the industry's standard dummy text ever since the 1500s, 
+                when an unknown printer took a standard dummy text.
+              </div>
+              <div class="grid-item navlang">
+                <a href="#" class="subgrid-item">CSS</a>
+                <img src="image/css-divider.png" alt="divider image" class="subgrid-item">
+                <a href="#" class="subgrid-item">html</a>
+                <img src="image/css-divider.png" alt="divider image" class="subgrid-item">
+                <a href="#" class="subgrid-item">bootstrap</a>
+                <img src="image/css-divider.png" alt="divider image" class="subgrid-item">
+                <a href="#" class="subgrid-item">Ruby</a>
+              </div>
+              <a href="#" class="grid-item seeproject">See Project</a>
+            </div>
+
+            <div class="project-grid">
+              <div class="grid-item"><img src="image/Img-Placeholder.png" alt="image of project"></div>
+              <div class="grid-item"><h2>Multi-Post Story</h2></div>
+              <div class="grid-item holdtext">
+                A daily selection of privately personalized reads; 
+                no accounts or sign-ups required. 
+                has been the industry's standard dummy text ever since the 1500s, 
+                when an unknown printer took a standard dummy text.
+              </div>
+              <div class="grid-item navlang">
+                <a href="#" class="subgrid-item">CSS</a>
+                <img src="image/css-divider.png" alt="divider image" class="subgrid-item">
+                <a href="#" class="subgrid-item">html</a>
+                <img src="image/css-divider.png" alt="divider image" class="subgrid-item">
+                <a href="#" class="subgrid-item">bootstrap</a>
+                <img src="image/css-divider.png" alt="divider image" class="subgrid-item">
+                <a href="#" class="subgrid-item">Ruby</a>
+              </div>
+              <a href="#" class="grid-item seeproject">See Project</a>
+            </div>
+
+          </div>
+        </section>
 
     </div>
   </body>
```

Comment:
- [ ] There is a lot of space after the work section. Fixing this will make your project look even better, I suggest you do that.

![image](https://user-images.githubusercontent.com/29071984/185744823-e6f149e6-482e-4dc4-8b78-d3efa55e6dce.png)